internal/pkg/cli/deploy/env.go

```diff
@@ -436,15 +437,18 @@ func (d *envDeployer) buildStackInput(in *DeployEnvironmentInput) (*cfnstack.Env
 }
 
 func (d *envDeployer) renderStaticSite(mft *manifest.Environment) error {
-	if mft == nil || mft.CDNConfig.Config.Static.Location.StaticSite == "" {
+	if mft == nil || mft.CDNConfig.Config.Static.Location.StaticBucket != "" ||
```

Comment:
plz excuse my ignorance - can `StaticBucket != ""` and `StaticSite != ""` be `true` at the same time, for the same location?

Reply:
i don't think so because they unmarshal from the same field https://github.com/aws/copilot-cli/blob/50989ce59f74fab653b6864c87dbb0141d37a0bd/internal/pkg/manifest/env.go#L298

Reply:
And `StaticBucket` is the rendered bucket URL value for `StaticSite`. I just don't want to overwrite the value for `StaticSite` as it's not an idempotent behavior